### DISCUSSION TITLE
关于这个算不算 Bug

### DISCUSSION THREAD

**Discussion by wuliaodexiaoluo** [Meloong-Git/PCL#4342] [提问] [answered]
- 更新失败后重新更新会再要求一遍密钥
日志：
[Log3.txt](https://github.com/user-attachments/files/16372266/Log3.txt)
[Log2.txt](https://github.com/user-attachments/files/16372267/Log2.txt)

 - 反复读取不存在的你知道吗的配置文件
 日志：
[Log3.txt](https://github.com/user-attachments/files/16372277/Log3.txt)

**Comment by WForst-Breeze** [comment, marked as answer]
前者个人不是很清楚，之前也遇到过，个人感觉不像（）可能是刻意的设计，不太确定

后者可能是预期外的行为，交一个 Issue？~~但是倒也没影响正常使用~~
